spec/controllers/items_controller_spec.rb

```diff
@@ -44,65 +44,114 @@
     end
 
     it "response redirect" do
-      post :create, :item => {name: "a", description: "A", price: 1, user_id: user.id}
+      post :create, :item => {name: "a", description: "A", buy_it_now_price: 1, user_id: user.id}
```

Comment:
we removed duplication by creating a let that specified the item_params for this context